serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -0,0 +1,32 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="upgrading-serverless"]
+= Upgrading {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: upgrading-serverless
+
+toc::[]
+
+If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest Generally Available version.
+
+In Technology Preview versions of {ServerlessProductName} (versions 1.6.0 and older), the only available Subscription Update Channel was `techpreview`.
+To upgrade to the latest Generally Available version, you must update the channel to the correct channel for your {product-title} version.
+
+.Prerequisites
+* You have installed a Technology Preview version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
+* If you have selected Manual updates for the {ServerlessOperatorName} during the installation process
+// what do we need to do here? Anything additional / different?
+
+.Procedure
+
+. Select the `openshift-operators` namespace in the {product-title} web console.
+. Navigate in the web console to the *Operators* → *Installed Operators* page.
+. Select the *{ServerlessOperatorName} Operator*.
+. Click *Subscription* → *Channel*.
+. In the *Change Subscription Update Channel* window, select the correct version for your {product-title} cluster version, and click *Save*.
+. Wait until all pods have been upgraded in the `knative-serving` namespace, and the `KnativeServing` custom resource reports the latest {ServerlessProductName} version.
```

Comment:
The KnativeServing doesn't report the Serverless version. It only reports the Knative Serving version (i.e. 0.13.1).